tornado_handlers.py

```diff
@@ -671,7 +672,71 @@ def get(self, *args, **kwargs):
         template = _ENV.get_template(BROWSE_TEMPLATE)
         self.write(template.render(table_data=table_header + table_data + table_footer))
 
+class DBInfoHandler(tornado.web.RequestHandler):
+    """ Get database info Tornado request handler """
 
+    def get(self, *args, **kwargs):
```

Comment:
At some point we should merge this into a common base class shared with the `BrowseHandler` to minimize code duplication, but for now it's ok.